### PR TITLE
Post for 2.2.1.Final

### PR DESCRIPTION
**Pull request by juangon**

### CODE REVIEW

File: src/posts/announce-2.2.1.Final.adoc [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+This release contains several bugfixes from different areas of the product, such as MicroProfile Config, MP RestClient or Topology used in OpenShift.
+MicroProfile OpenAPI now works using Swarm.run() from IDE.
+There were other improvements around dependencies in some fractions.
+
+++++
+<!-- more -->
+++++
+
+== Deprecation and config keys breaking changes
```
Comment (Ladicek) [inline src/posts/announce-2.2.1.Final.adoc]: I would split this into two. Deprecating `@CreateSwarm` has nothing in common with the `swarm.*` to `thorntail.*` change.

File: src/posts/announce-2.2.1.Final.adoc [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+title: Announcing Thorntail 2.2.1.Final
+publishDate: 2018-10-30
+modifyDate: 2018-10-30
+author: The Thorntail Team
+layout: post.jade
+---
+
+== Thorntail 2.2.1.Final
+
+Winter is coming! (at least in the Northern Hemisphere) but we can stay warm with another Thorntail release! Say "hello" to Thorntail 2.2.1.Final!
```
Comment (Ladicek) [inline src/posts/announce-2.2.1.Final.adoc]: I don't think the punctuation here is proper English :-)

**Comment by juangon** [comment]
@Ladicek I made your suggested changes. If that's fine I'll rebase into a single commit to make history more clear

File: src/posts/announce-2.2.1.Final.adoc [hunk 3]
```diff
@@ -24,11 +24,16 @@ There were other improvements around dependencies in some fractions.
 <!-- more -->
 ++++
 
-== Deprecation and config keys breaking changes
+== @CreateSwarm is deprecated
 
 *WARNING: Breaking changes!*
 
 @CreateSwarm annotation is now deprecated and shouldn't be used anymore.
```
Comment (Ladicek) [inline src/posts/announce-2.2.1.Final.adoc]: Add the "@CreateSwarm" bit into backticks, so that it's highlighted as code?

File: src/posts/announce-2.2.1.Final.adoc [hunk 4]
```diff
@@ -24,11 +24,16 @@ There were other improvements around dependencies in some fractions.
...
+
+== Config keys breaking changes
+
+*WARNING: Breaking changes!*
+
 As an additional task on renaming from Wildfly Swarm to Thorntail, all configuration keys had been renamed from swarm.x.y to thorntail.x.y.
```
Comment (Ladicek) [inline src/posts/announce-2.2.1.Final.adoc]: Wildfly -> WildFly

Also add "swarm.x.y" and "thorntail.x.y" into backticks.

**Comment by juangon** [comment]
@Ladicek done

File: src/posts/announce-2.2.1.Final.adoc [hunk 5]
```diff
@@ -24,11 +24,16 @@ There were other improvements around dependencies in some fractions.
...
 
 @CreateSwarm annotation is now deprecated and shouldn't be used anymore.
+
+== Config keys breaking changes
+
+*WARNING: Breaking changes!*
+
 As an additional task on renaming from Wildfly Swarm to Thorntail, all configuration keys had been renamed from swarm.x.y to thorntail.x.y. 
 It's recommended to update those keys if used although previous names should work as well.
 Unfortunately, there is a known issue when using old key names in System Properties (see https://issues.jboss.org/browse/THORN-2205[THORN-2205]), so just change those System Properties with the new name.
```
Comment (Ladicek) [inline src/posts/announce-2.2.1.Final.adoc]: I'd reword the entire sentence to something like:

> The old `swarm.` prefix no longer works for certain system properties.
> See https://issues.jboss.org/browse/THORN-2205[THORN-2205] for more details.
> It is recommended to change all occurences of `-Dswarm.*` in all Thorntail invocations, including the Thorntail Maven plugin, to `-Dthorntail.*`.

Comment (juangon) [reply src/posts/announce-2.2.1.Final.adoc]: @Ladicek done

**Review by Ladicek — APPROVED** [review]

**Comment by juangon** [comment]
Ok thanks for revewing it. I rebased onto a single commit
